### ISSUE TITLE
specutils 1.5 breaks DRAGONS 3.0.x (and master)

### ISSUE THREAD

**Issue by KathleenLabrie** [GeminiDRSoftware/DRAGONS#312]
Just a note so that we don't forget.  It was found today (by Chris) that specutils 1.5 breaks 3.0.x.  While that gets sorted out (probably a big job), the workaround for users is to revert to specutils v1.1. 

v1.5 is currently only available on conda-forge.  We tell the users to use the astroconda channel and that has only 1.1, so that mitigates the problem a bit.

**Comment by kelle** [comment]
Is this still true? Does current DRAGONS work with latest specutils? If not, what's the breaking change?

**Comment by jehturner** [comment]
No, I believe this was resolved some time ago (by no longer relying on `Spectrum1D`, I think), but I'll let @KathleenLabrie confirm that she's happy to close the issue.

**Comment by kelle** [comment]
Ah, glad to hear it's resolved but sad to hear the solution.

**Comment by jehturner** [comment]
I think it was being used in an incidental fashion, where it didn't make a lot of sense to be vulnerable to upstream API changes. I fully expect that we would use it in future if we were to produce 1D spectra intended to be compatible with other packages.
